framework-tests/app/src/androidTest/java/org/gearvrf/tester/PickerTests.java

```diff
@@ -690,4 +693,49 @@ public void pickQuadFromLeftAndRight()
         mPickHandler.checkNoHits("quad1");
         mPickHandler.checkNoHits("quad2");
     }
+
```

Comment:
When picking from an object, the pick ray should be (0,0,0) to (0,0,1). I have made the picker default to this in the new constructor and now the demo works. This is pull request #1145